src/adapter/src/coord/ddl.rs

```diff
@@ -62,7 +62,7 @@ impl<S: Append + 'static> Coordinator<S> {
 
         let mut sources_to_drop = vec![];
         let mut tables_to_drop = vec![];
-        let mut sinks_to_drop = vec![];
+        let mut storage_sinks_to_drop = vec![];
```

Comment:
Maybe I don't fully understand the new model, but why isn't there any compute sinks?

Reply:
`catalog::Sink`s are all storage sinks!  Things like PersistSink (which live in Compute) are an internal implementation detail for things like materialized views